source/docs/partnering/experts/spec.yml

```diff
@@ -1,12 +1,11 @@
-swagger: "2.0"
 info:
-  version: "1.0.0"
+  version: 1
   title: Expert APIs
   description: |-
-    A collection of APIs providing data about national laboratorty experts related to associated with the Department of Energy's (DOE) Lab Partnering Service (LPS).
+    A collection of APIs providing data about national laboratorty experts associated with the Department of Energy's (DOE) Lab Partnering Service (LPS).
 schemes:
   - https
-basePath: /api/lps/v1
+basePath: /api/v1/experts
```

Comment:
Are these paths correct? I think it should maybe be `/api/lps/v1/experts`, since nothing is at https://developer.nrel.gov/api/v1/experts

Reply:
These paths are correct, we're pushing updates to the prod version of the underlying API scheduled for today.